tests/integration/test_permissions.py

```diff
@@ -58,6 +58,7 @@ def _create_and_index_record(record):
 def sample_record(app):
     record = {
         '$schema': 'http://localhost:5000/schemas/records/hep.json',
+        'authors': [],
```

Comment:
Obviously this is because this PR adds a check that there must be either an `authors` or a `corporate_author`. But then https://github.com/inspirehep/inspire-schemas/pull/105 will break this test, so can we future-proof this?